[BUG] Integration test failures using Ollama models to run langchain4j-agentic

**Describe the bug**
Encountered errors/failures when running langchain4j-agentic integration tests

**To Reproduce**
```
cd langchain4j-agentic
```

```
# change to using OLLAMA MODEL_PROVIDER
vi src/test/java/dev/langchain4j/agentic/Models.java
```

change 
```
private static final MODEL_PROVIDER modelProvider = Models.MODEL_PROVIDER.OPENAI;
```
to
```
private static final MODEL_PROVIDER modelProvider = Models.MODEL_PROVIDER.OLLAMA;
```

```
mvn clean
mvn verify
```

notice test failures:
```
[ERROR] Errors: 
[ERROR]   SupervisorAgentIT.agentic_banker_test:206 » Timeout request timed out
[ERROR]   SupervisorAgentIT.agentic_banker_with_agentic_exchange_test:276->agentic_banker_with_exchange_test:290->agentic_banker_with_exchange_test:327 » Timeout request timed out
[ERROR]   SupervisorAgentIT.agentic_banker_with_non_agentic_exchange_test:281->agentic_banker_with_exchange_test:290->agentic_banker_with_exchange_test:327 » Timeout request timed out
[ERROR]   SupervisorAgentIT.agents_system_test:91 » Timeout request timed out
```


**Expected behavior**
should see tests all successful:
```
[INFO] Tests run: 28, Failures: 0, Errors: 0, Skipped: 0
```

**Please complete the following information:**
- LangChain4j version: 1.4.0-SNAPSHOT
- LLM(s) used: qwen2.5:7b, qwen3:8b
- Java version: 17
- Spring Boot version (if applicable): NA

--- a/langchain4j-agentic/src/main/java/dev/langchain4j/agentic/supervisor/PlannerAgent.java
+++ b/langchain4j-agentic/src/main/java/dev/langchain4j/agentic/supervisor/PlannerAgent.java
@@ -10,26 +10,26 @@ public interface PlannerAgent {
     @SystemMessage("""
             You are a planner expert that is provided with a set of agents.
             You know nothing about any domain, don't take any assumptions about the user request,
-            the only thing that you can do is relying on the provided agents.
+            the only thing that you can do is rely on the provided agents.
             
-            Your role is to analyze the user request and decide which of the provided agent to call next to address it.
-            You return an agent invocation containing the name of the agent and the arguments to pass to it.
-            Generate the agent invocation also considering the past messages and in the same language of the user request.
-            If the user request have been already completely fulfilled or the last response already contains an
-            appropriate answer, simply return an agent invocation with agentName "done" and a single argument named
-            "response" containing the answer to be returned to the user with a recap of all the performed actions
-            written in the same language of the user request.
+            Your role is to analyze the user request and decide which one of the provided agents to call next to address it.
+            You return an agent invocation consisting of the name of the agent and the arguments to pass to it.
             
-            For each agent it will be provided both the name and description together with the list of the arguments
-            it takes in input using the format {name: description, [argument1, argument2]}.
-            """)
-    @UserMessage("""
+            If no further agent requests are required, return an agentName of "done" and an argument named
+            "response", where the value of the response argument is a recap of all the performed actions,
+            written in the same language as the user request.
+
+            Agents are provided with their name and description together with a list of applicable arguments
+            in the format {name: description, [argument1, argument2]}.
+
             Decide which agent to invoke next, doing things in small steps and
             never taking any shortcuts or relying on your own knowledge.
-            Even if the user's request is already clear or explicit, don't make any assumption and use all agents.
-            You MUST query ALL necessary agents.
+            Even if the user's request is already clear or explicit, don't make any assumptions and use the agents.
+            Be sure to query ALL necessary agents.
 
             The comma separated list of available agents is: '{{agents}}'.
+            """)
+    @UserMessage("""
             The user request is: '{{request}}'.
             The last received response is: '{{lastResponse}}'.
             """)

--- a/langchain4j-agentic/src/test/java/dev/langchain4j/agentic/Models.java
+++ b/langchain4j-agentic/src/test/java/dev/langchain4j/agentic/Models.java
@@ -48,6 +48,7 @@ public class Models {
             .temperature(0.0)
             .logRequests(true)
             .logResponses(true)
+            .think(false)
             .build();
 
     public static ChatModel baseModel() {

--- a/langchain4j-agentic/src/test/java/dev/langchain4j/agentic/SupervisorAgentIT.java
+++ b/langchain4j-agentic/src/test/java/dev/langchain4j/agentic/SupervisorAgentIT.java
@@ -104,24 +104,24 @@ public class SupervisorAgentIT {
 
     public interface WithdrawAgent {
         @SystemMessage("""
-            You are a banker that can only withdraw US dollars (USD) from a user account,
+            You are a banker that can only withdraw US dollars (USD) from a user account.
             """)
         @UserMessage("""
-            Withdraw {{amount}} USD from {{user}}'s account and return the new balance.
+            Withdraw {{amountInUSD}} USD from {{user}}'s account and return the new balance.
             """)
         @Agent("A banker that withdraw USD from an account")
-        String withdraw(@V("user") String user, @V("amount") Double amount);
+        String withdraw(@V("user") String user, @V("amountInUSD") Double amount);
     }
 
     public interface CreditAgent {
         @SystemMessage("""
-            You are a banker that can only credit US dollars (USD) to a user account,
+            You are a banker that can only credit US dollars (USD) to a user account.
             """)
         @UserMessage("""
-            Credit {{amount}} USD to {{user}}'s account and return the new balance.
+            Credit {{amountInUSD}} USD to {{user}}'s account and return the new balance.
             """)
         @Agent("A banker that credit USD to an account")
-        String credit(@V("user") String user, @V("amount") Double amount);
+        String credit(@V("user") String user, @V("amountInUSD") Double amount);
     }
 
     static class BankTool {

